base/all/data-raw/record_package_versions_for_release.R

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env Rscript
+
+# Usage:
+# ./base/all/data-raw/record_package_versions_for_release.R [tag ...] [save]
+#
+# START WITH A CLEAN GIT TREE. This script checks out tags to read their
+# versions. It complains before clobbering anything and restores state
+# after run, but no warranties etc
+#
+# No "save" => just write to console
+# ./base/all/data-raw/record_package_versions_for_release.R v1.2.3
+#
+# With "save" => add results to package data files
+# base/all/data/pecan_version_history.rda & base/all/data/pecan_releases.rda
+# ./base/all/data-raw/record_package_versions_for_release.R v4.5.6 v4.5.7 save
```

Comment:
When I wrote this I was picturing that we could use this to add new versions at release time, but that implies creating the tags before committing the updated data files! Probably need an option to report the versions at HEAD and write them to file with a specified tag name.

Reply:
Probably better yet:
* Change this script to report only date rather than datetime in `pecan_releases` table
* rerun and save result
* leave this script as a historical, overly-baroque artifact
* write a new script that only
    - takes the desired tag name, version, and release date as arguments
    - writes the tagname, version, and release date as a new row in `pecan_versions`
    - calls the guts of `scan_at_tag` without performing any Git operations (i.e. scans at HEAD)
    - writes the result to a new column of `pecan_version_history`